### PR TITLE
Afd 90 edit book view

### PR DESCRIPTION
**Pull request by TheGuardingDark**
Finished styling for all pages with breakpoints and began working on mobile menu.

Known issues: 
mobile menu doesn't close when rendering new page. hamburger gets stuck in 'open position.

blank space between content and footer at smaller desktop sizes, almost always at 769px

### CODE REVIEW

**Review by katieburrows — DISMISSED** [review]
I love the hamburger menu animation!  On my system it looks like the block is moving too far up and to the left and then re-centers in a glitchy kind of way.  It starts out smooth then gets a little glitchy.  I'm also noticing extra space on the right-hand side of the navbar when I'm at 480px.

File: src/components/BookForm.jsx [hunk 1]
```diff
@@ -1,26 +1,43 @@
...
 import EmptyCard from './EmptyCard';
 import Button from './Button';
 import Stars from './Stars';
 
-const BookForm = () => {
+const BookForm = ({ src, title, author, synopsis, published, pages }) => {
   return (
     <form className="form grid form__mobile" id="addBookForm">
-      <label className="form__text form__mobile" htmlFor="title">
+      <label
```
Comment (scullenDev) [inline src/components/BookForm.jsx]: To placate your linter, you'll have to both wrap the labels AROUND their respective inputs **and** still associate the label with the input via the `httpFor` attribute. Let me know if you have related questions.

File: src/components/Button.jsx [hunk 2]
```diff
@@ -8,6 +10,13 @@ const Button = ({ text, type, className }) => {
 
 Button.defaultProps = {
```
Comment (scullenDev) [inline src/components/Button.jsx:11]: Thank you for doing your propTypes/defaultProps up front... yay!

File: src/components/CustomPub.jsx [hunk 3]
```diff
@@ -14,7 +14,7 @@ const CustomPub = () => {
           form__mobile
           form__selectPub--input"
       >
-        <option value="0" />
+        <option value="0">{date}</option>
```
Comment (scullenDev) [inline src/components/CustomPub.jsx]: I like to believe that all books under the sun were published on one of these four dates.

**Review by scullenDev — CHANGES_REQUESTED** [review]
Requesting minor changes.

Is there still code you want me to look at re: the hamburger menu? If so, please send me a branch name to work off of and a link to the documentation for the package you used.

File: src/components/CustomPub.jsx (hunk 3 above)
Comment (TheGuardingDark) [reply src/components/CustomPub.jsx]: They were, Sarah. They were. But yeah that'll change once the edit form is wired up

**Review by scullenDev — APPROVED** [review]
Looks good!

**Review by aperezsantos — APPROVED** [review]
Beyond the known issues and some of the commented code, which I assume are things you're going to work on, I think everything looks great.

**Pull request merged**